src/aead/aead.rs

```diff
@@ -296,23 +297,51 @@ mod tests {
     }
 
     #[test]
+    #[ignore]
+    pub fn test_aes_gcm_128_slow() {
+        test_aead_slow(&aead::AES_128_GCM,
```

Comment:
Similarly, use a more descriptive suffix for this.

("_slow" may be descriptive, but it doesn't hint at the reason we wrote these tests.)